index.html

```diff
@@ -2408,7 +2410,7 @@ <h3>Human-Readable Metadata</h3>
 <pre class="example">
 {
     "@context": "https://www.w3.org/2019/wot/td/v1",
-    "title": "MyThing",
+    "title": "MeinDing",
```

Comment:
Do not quite understand why it is changed to German. Having said that, I think both is fine.

Reply:
Keeping this in English would confuse reader since the setting is "@language" : "de"

Reply:
I think you changed the **wrong** example.

You changed [Example 7](https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/989.html#example-7) and wanted to change [Example 8](https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/989.html#example-8).
Is this the case?

Reply:
thanks. last commit should be correct now